stories/s2l1c.stories.js

```diff
@@ -54,7 +54,7 @@ export const GetMapWMS = () => {
 
   // getMap is async:
   const perform = async () => {
-    const layerS2L1C = new S2L1CLayer(instanceId, s2l1cLayerId);
+    const layerS2L1C = new S2L1CLayer({ instanceId, layerId: layerId });
```

Comment:
`layerId: layerId` could be changed to just `layerId`

Reply:
Fixed.

Reply:
Great, resolving.